src/org/camunda/latera/bss/connectors/hoper/Hoper.groovy

```diff
@@ -18,12 +18,11 @@ class Hoper {
     def ENV     = System.getenv()
 
     this.url      =  execution.getVariable("hoperUrl")      ?: ENV['HOPER_URL']     ?: 'http://hoper:3000'
-    this.version  = (execution.getVariable("hoperVersion")  ?: ENV['HOPER_VERSION'] ?: 2)?.toInteger()
```

Comment:
Please remove check of version here:
https://github.com/latera/camunda-ext/blob/master/src/org/camunda/latera/bss/connectors/hoper/hydra/Company.groovy#L4-L20

Without that company methods in hoper.Hydra become broken.